notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        Args:
                    save_tensor_policy: Users can pass a callback with signature fn(offloaded_tensors, forward_trace) to filter
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>